admin/netdata/patches/005-patch-collectors_python.d.plugin_python__modules_bases_loaders.py.patch

```diff
@@ -0,0 +1,14 @@
+--- a/collectors/python.d.plugin/python_modules/bases/loaders.py
++++ b/collectors/python.d.plugin/python_modules/bases/loaders.py
```

Comment:
Do you have any plans to upstream it or at least to talk to them about that? I am not sure, why they have in their repository `pyyaml2` and `pyyaml3`.  I see that this patch has many distributions.

Reply:
Most likely because they want to workaround package managers for whatever reason (just look at the "install script") in upstream repo

Reply:
This is such bad... I will create an issue and ask why they vendor pyyaml and so on. It does not make any sense.